public/editor.js

```diff
@@ -1039,9 +1039,20 @@ class UINode
 
         let portDiv = document.createElement('div');
         portDiv.className = (side == 'dst')? 'node_in_port':'node_out_port';
-        portDiv.onpointerdown = (evt) => evt.stopPropagation();
-        portDiv.onpointerup = (evt) => evt.stopPropagation();
-        portDiv.onclick = portClick.bind(this);
+
+        portDiv.onpointerdown = (evt) => {
+          evt.stopPropagation();
+          portDiv.setPointerCapture(evt.pointerId);
+          portClick.call(this, evt);
+        }
+
+        portDiv.onpointerup = (evt) => {
+          evt.stopPropagation();
+          portDiv.releasePointerCapture(evt.pointerId);
+        }
+
+        portDiv.onclick = (evt) => evt.stopPropagation();
```

Comment:
If I understand correctly, you're using setPointerCapture here to avoid triggering an onclick or onpointerup on the editor. I'm ok with that.

Would this still work if the portClick call occurs in portDiv.onclick though?

i.e. can you keep `portDiv.onclick = portClick.bind(this);` ?

Reply:
@maximecb 
Yeah, you could just keep `portDiv.onclick = portClick.bind(this);` instead of where I placed in on ln1046.

The difference is that this will only start drawing the wire once you release the mouse button, where the version above starting drawing it as soon as you click.

Here is a PR for that version to compare the feel of: 
https://github.com/maximecb/noisecraft/pull/69